plugins/callback/opentelemetry.py

```diff
@@ -560,7 +560,12 @@ def dump_results(self, result):
         """ dump the results if disable_logs is not enabled """
         if self.disable_logs:
             return ""
-        return self._dump_results(result._result)
+        # ansible.builtin.uri contains the response in json format
+        # let's remove the json field to avoid issues when the response is huge.
+        save = result._result
+        if "json" in save:
```

Comment:
A `json` field could also be returned by various other modules. Either there should be a more extensive check to make sure this only covers the `uri` module, or you have to update the changelog fragment to say that it removes all `json` return values from all modules/actions, or you add some size check to only remove it if it's huge.

Reply:
(There are also other modules which can return huge results, for example ansible.builtin.slurp.)

Reply:
Thanks Felix, I'll see If I can filter by

Reply:
see https://github.com/ansible-collections/community.general/pull/8430/commits/52b64be6be1030a8b43fbdc93073fbfe52982754